packages/react-styled-ui/src/TMIcon/tmIconSvgMap.js

```diff
@@ -0,0 +1,2209 @@
+  }, {
+    code: 'e0d6',
+    iconset: 0,
+    name: 'wmi',
+    new: true,
+    paths: ['M960 256h-256v-256zM640 320h320v666.664c-0.367 20.699-17.233 37.338-37.985 37.338-0.146 0-0.291-0.001-0.437-0.002l0.022 0h-819.2c-0.123 0.001-0.269 0.002-0.415 0.002-20.752 0-37.618-16.639-37.985-37.304l-0-0.034v-949.332c0.366-20.697 17.23-37.334 37.98-37.334 0.148 0 0.295 0.001 0.443 0.003l-0.022-0h537.6zM768 640l-89.6 179.2 57.6 25.6 102.4-204.8-102.4-204.8-57.6 25.6zM352 819.2l-89.6-179.2 83.2-179.2-57.6-25.6-102.4 204.8 108.8 204.8zM608 390.4l-64-12.8-128 512 64 12.8z']
+  }, {
+    code: 'e0d0',
+    iconset: 0,
+    name: 'yoda',
+    new: false,
+    paths: ['M262.4 505.6c-96 25.6-172.8 102.4-198.4 198.4l-64 256h1024l-64-256c-25.6-96-102.4-172.8-198.4-198.4-19.2-6.4-38.4-12.8-51.2-25.6 19.2-25.6 38.4-64 44.8-96v0c64 6.4 128-25.6 160-76.8 0 0 25.6-64 76.8-76.8 32-6.4 19.2-25.6 19.2-25.6-19.2-19.2-51.2-19.2-115.2 0-51.2 6.4-108.8 6.4-153.6-12.8-44.8-102.4-128-192-230.4-192s-185.6 89.6-230.4 192c-51.2 19.2-102.4 19.2-153.6 12.8-64-19.2-96-19.2-121.6-6.4 0 0-12.8 12.8 19.2 25.6 51.2 12.8 76.8 76.8 76.8 76.8 32 57.6 96 89.6 160 83.2v0c6.4 38.4 25.6 70.4 51.2 96-19.2 12.8-32 25.6-51.2 25.6z']
+  }, {
+    code: 'e0b0',
+    iconset: 0,
+    name: 'zoom-in',
+    new: false,
+    paths: ['M748.8 659.2c134.4-185.6 96-448-89.6-582.4s-448-96-582.4 89.6-96 448 89.6 582.4c147.2 108.8 345.6 108.8 492.8 0l275.2 275.2 89.6-89.6-275.2-275.2zM416 736c-179.2 0-320-140.8-320-320s140.8-320 320-320 320 140.8 320 320c0 179.2-140.8 320-320 320zM448 384h192v64h-192v192h-64v-192h-192v-64h192v-192h64v192z']
+  }, {
+    code: 'e0b1',
+    iconset: 0,
+    name: 'zoom-out',
+    new: false,
+    paths: ['M748.8 659.2c134.4-185.6 96-448-89.6-582.4s-448-96-582.4 89.6-96 448 89.6 582.4c147.2 108.8 345.6 108.8 492.8 0l275.2 275.2 89.6-89.6-275.2-275.2zM416 736c-179.2 0-320-140.8-320-320s140.8-320 320-320 320 140.8 320 320c0 179.2-140.8 320-320 320zM192 384h448v64h-448c0 0 0-64 0-64z']
+  }]
+};
+
+const tmIconSvgMap = tmIconMap.icons.reduce((prevMap, { name, paths }) => {
+  return {
+    ...prevMap,
+    [`tmicon-${name}`]: {
+      path: (
+        <g>
+          {paths.map((path, idx) => (
+            <path
+              key={`${name}-${idx}`}
+              fill="currentColor"
+              d={path}
+            />
+          ))}
+        </g>
+      ),
+      viewBox: '0 0 1060 1060',
```

Comment:
@joshuatai Can you help to double confirm that the exact viewBox we can use? If we use '0 0 960 960' some of the icon's right and bottom edge will be cropped.

Reply:
Never mind, I will import icon with `0 0 16 16` and we can ignore this problem :)

Reply:
Part of the icon will use `0 0 1060 1060` as a workaround, will be tackled in a separate PR.